### DISCUSSION TITLE
curl-distros Mailing list 404's

### DISCUSSION THREAD

**Discussion by christian-heusel** [curl/curl#13833] [General]
Hey, while I'm subscribed myself I think it would be cool if the "curl-distros" mailing list had public archives: 

https://lists.haxx.se/listinfo/curl-distros ➡️ "visit the [curl-distros Archives](https://lists.haxx.se/pipermail/curl-distros/)." ➡️  HTTP404

https://lists.haxx.se/pipermail/curl-distros/

**Comment by bagder** [comment]
The archive is available [here](https://curl.se/mail/list.cgi?list=curl-distros), it is just the list page that is still wrong...

**Reply by christian-heusel** [reply]
Ah great, thanks for the pointer! 😊

**Reply by christian-heusel** [reply]
Added https://github.com/curl/curl/pull/13843 to include it in the `DISTROS.md` document 😄
